app/authorization/server/methods/saveRole.js

```diff
@@ -19,6 +20,12 @@ Meteor.methods({
 				method: 'authorization:saveRole',
 			});
 		}
+		const roleExists = Roles.findOne(roleData.name);
```

Comment:
Since you're just checking for the existance of the role, add a `{ projection: { _id: 1 } }` as options to the call to findOne so the app doesn't have to fetch the whole thing